spec/services/form_response_spec.rb

```diff
@@ -56,6 +60,42 @@
       expect(combined_response.errors).to eq(front: 'error', back: 'error')
     end
 
+    it 'merges multiple errors for key' do
+      response1 = FormResponse.new(success: false, errors: { front: 'front-error-1' })
+      response2 = IdentityDocAuth::Response.new(success: true, errors: { front: ['front-error-2'] })
+
+      combined_response = response1.merge(response2)
+      expect(combined_response.errors).to eq(front: ['front-error-1', 'front-error-2'])
+    end
```

Comment:
FYI I'd expect this test case to pass on `main` as well, but it doesn't:

```
Failures:

  1) FormResponse#merge merges multiple errors for key
     Failure/Error: expect(combined_response.errors).to eq(front: ['front-error-1', 'front-error-2'])
     
       expected: {:front=>["front-error-1", "front-error-2"]}
            got: {:front=>["front-error-2"]}
     
       (compared using ==)
     
       Diff:
       @@ -1 +1 @@
       -:front => ["front-error-1", "front-error-2"],
       +:front => ["front-error-2"],
       
     # ./spec/services/form_response_spec.rb:56:in `block (3 levels) in <top (required)>'
```

So we incidentally fix an issue where we may be dropping some errors in merging two responses.